### ISSUE TITLE
Video Messages, Voice messages and BBB not working

### ISSUE THREAD

**Issue by Nonabz** [RocketChat/Rocket.Chat#22535]
I install a fresh BBB Server. Direct use success
install fresh rocket Chat Server on debain, settings Video Conf I add the datas from the bbb-conf --secret

`# bbb-conf --secret

    URL: https://chat.XXXXXXXXXX.com/bigbluebutton/
    Secret: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx
`

there is also a Link to the API-Mate:
    https://mconf.github.io/api-mate/#server=https://XXXXXXXXXXXXXXX 
dont know what this is 

I reg some user and try to start Video conf. I click on Start Meeting but nothing happens.

Also if i Record Voice or Video messages and send it, cannot play. Android Firefox say unsupported format

hope can help me :)

**Comment by yashovardhan** [comment]
Hey there,

Thanks a lot for reporting this!

Please follow our guidelines for reporting issues:

https://rocket.chat/docs/contributing/reporting-issues

Bugs are often fixed on the latest release so please test it there first and see if it still occurs. You can also test on the development version, or try on open.rocket.chat which is usually the latest development code.


You can ask for [support](https://docs.rocket.chat/getting-support) here:

https://open.rocket.chat
https://forums.rocket.chat

**Comment by Karting06** [comment]
@Nonabz You need to install `bbb-webhooks` (`sudo apt-get install bbb-webhooks`).

**Comment by Nonabz** [comment]
Thanx, i geht it to run already by installing the webhooks and remive the /bigbluebutton in the URL 
URL: https://chat.XXXXXXXXXX.com/bigbluebutton/
TO: URL: https://chat.XXXXXXXXXX.com/

But in the small video window (iframe?) I see tiny little my can and possibly that of the chat partner. Unfortunately it is almost impossible to click on something. 

Technically it works but for the normal user unfortunately (yet) not to use. 

Can I make settings so that I immediately see the picture of my chat partner without much clicking around?

**Comment by Karting06** [comment]
You can't change the size. But there is an option to make it full screen. 
Click on the three dots (inside the iframe), and then the first option `full screen`.
![image](https://user-images.githubusercontent.com/1096435/124626840-513b8480-de7f-11eb-84c9-3c5730f67441.png)

**Comment by Nonabz** [comment]
Yes, i can do no Problem. But the Clients don't know this and never read manuals. :( 

Maybe is a idea to implemented matrix synapse. The video options of this system are great. But I'm not a coder :(
